breadcrumb/src/Crumb.jsx

```
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export default function Crumb({ title, href, active, styles }) {
+  const classes = cx(styles.crumb, { [styles.active]: active });
+  if (active) {
+    return (<span className={classes}>{title}</span>);
+  }
+
+  return (<a href={href} className={classes}>{title}</a>);
+}
+
+Crumb.propTypes = {
+  title: PropTypes.string.isRequired,
+  href: PropTypes.string,
+  active: PropTypes.bool,
+  styles: PropTypes.object
```

Should this be required?
Won't this error if no styles are passed in?

---

It would but since I am attaching the styles prop to the Crumb in Breadcrumb through `React.cloneElement`, it should always have them. Suggestions welcome on a better way to do this, but since on the implementer side styles aren't required, I didn't want to throw an error